Dockerfile

```diff
@@ -0,0 +1,41 @@
+# AlpineLinux with a glibc-2.23, Oracle Java 8, sbt and git
+FROM anapsix/alpine-java:8_jdk
+
+# sbt
+
+ENV SBT_URL=https://dl.bintray.com/sbt/native-packages/sbt
+ENV SBT_VERSION 0.13.15
+ENV INSTALL_DIR /usr/local
+ENV SBT_HOME /usr/local/sbt
+ENV PATH ${PATH}:${SBT_HOME}/bin
+
+# Install sbt
+RUN apk add --no-cache --update bash wget && mkdir -p "$SBT_HOME" && \
+    wget -qO - --no-check-certificate "https://dl.bintray.com/sbt/native-packages/sbt/$SBT_VERSION/sbt-$SBT_VERSION.tgz" |  tar xz -C $INSTALL_DIR && \
+    echo -ne "- with sbt $SBT_VERSION\n" >> /root/.built
+
+# Install git
+RUN apk add --no-cache git openssh
+
+# Install node.js
+RUN apk add nodejs
```

Comment:
should we specify a version?

Reply:
Done.

Reply:
reviewed alpine recommendations, they suggest not doing this unless you've proven it's needed - the packages are regularly dropped from the repo and dockerfile builds will fail.

https://gitlab.alpinelinux.org/alpine/abuild/-/issues/9996#note_87135

Reply:
thanks for looking more closely at this!